backend/users/tests/test_social_link_rewards.py

```
@@ -0,0 +1,85 @@
+from social_connections.models import DiscordConnection, TwitterConnection
+
+User = get_user_model()
+
+
+class SocialLinkRewardTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects.create_user(
+            email='social-link@test.com',
+            address='0x1234567890123456789012345678901234567890',
+            password='testpass123',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)